titan-server/src/catchers.rs

```diff
@@ -1,30 +1,38 @@
 use rocket_contrib::json::Json;
-use crate::api::ApiResponseBody;
+use crate::api::ApiErrorResponseBody;
 
 #[catch(400)]
-pub fn handle_400_bad_request(_: &rocket::Request) -> Json<ApiResponseBody> {
-    Json(ApiResponseBody {
-        message: "Invalid or malformed input".to_string(),
+pub fn handle_400_bad_request<'r>(_: &rocket::Request) -> Json<ApiErrorResponseBody> {
```

Comment:
The lifetime annotation shouldn't be necessary here. I think we can remove it.

Reply:
Removed. The need for a lifetime specifier has been refactored out (`message` used to be of type `&str`)